src/infra/flow-system-inmem/tests/tests/test_schedule.rs

```diff
@@ -0,0 +1,58 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use std::assert_matches::assert_matches;
+
+use chrono::prelude::*;
+use chrono::Utc;
+use kamu_flow_system::{
+    CronExpressionIterationError,
+    InvalidCronExpressionError,
+    Schedule,
+    ScheduleError,
+};
+
+#[test]
```

Comment:
Maybe, convert this to traditional unit test module located in `Schedule` file ?